com.playeveryware.eos/Runtime/Core/Config/Config.cs

```diff
@@ -220,7 +220,7 @@ public static T Get<T>() where T : Config
         /// <returns>
```

Comment:
Instead of being made `public`, if these things need to be utilized outside of the `Config` class, I believe that they should be moved to a shared location outside of where either is used. Perhaps they can be moved into the `EnumUtility` class itself?

Reply:
It is reverted instead because it isn't used anywhere in the plugin